cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -1069,6 +1137,11 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
                         name: 'visibility',
                         displayName: gettext('Visibility'),
                         editors: []
+                    },
+                    {
+                        name: 'discussion',
+                        displayName: gettext('Discussion'),
+                        editors: [DiscussionEditor]
```

Comment:
Per earlier conversation, would this block go away?
It looks like it's what's used to render the UI for non-verticals...

Reply:
I'll get back to this comment after getting some more context on the other PR next week.